aries-site/src/data/wcag/datatable.json

```diff
@@ -0,0 +1,518 @@
+  },
+  {
+    "rule": "3.1.1",
+    "label": "Language of Page",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.2",
+    "label": "Language of Parts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.3",
+    "label": "Unusual Words",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.4",
+    "label": "Abbreviations",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.5",
+    "label": "Reading Level",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.6",
+    "label": "Pronunciation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.1",
+    "label": "On Focus",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.2",
+    "label": "On Input",
+    "status": "failed",
+    "notes": "re select checkboxes ok but search silently changes contents of table"
+  },
+  {
+    "rule": "3.2.3",
+    "label": "Consistent Navigation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.4",
+    "label": "Consistent Identification",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.5",
+    "label": "Change on Request",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.6",
+    "label": "Consistent Help",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.1",
+    "label": "Error Identification",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.2",
+    "label": "Labels or Instructions",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.3",
+    "label": "Error Suggestion",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.4",
+    "label": "Error Prevention (Legal, Financial, Data)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.5",
+    "label": "Help",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.6",
+    "label": "Error Prevention (All)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.7",
+    "label": "Redundant Entry",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.8",
+    "label": "Accessible Authentication (Minimum)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.9",
+    "label": "Accessible Authentication (Enhanced)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "4.1.2",
+    "label": "Name, Role, Value",
+    "status": "failed",
+    "notes": "no appropriate hint/role for column resize, aria-sort for ascending/descending"
+  },
+  {
+    "rule": "4.1.3",
+    "label": "Status Messages",
+    "status": "failed",
+    "notes": "rows added or removed due to search control or infinite scroll aren't announced. Rows added by group expand and/or row details expand"
```

Comment:
This is an issue but I don't think it is related to status messages. I would put this under info and relationships

Reply:
okay makes sense here

Reply:
so then datatable doesn't have anything applicable here if this is more for errors and success pop-ups?

Reply:
okay yeah originally was thinking the same but then as a group we wernt sure if the other stuff we mentioned in notes would count here